.github/workflows/pyside6.yml

```diff
@@ -0,0 +1,34 @@
+name: Run PySide6 Tests
+
+on:
+  push:
+    branches:
+      - "*"
+  pull_request:
+    branches:
+      - "*"
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-latest
+            VFXPLATFORM: "2018"
+            PYTHON: "3.6"
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: pip install PySide6
+        run: |
+          pip install PySide6
+
+      - name: Run tests in Docker container
+        run: |
+          docker run --rm \
+            -v $(pwd):/Qt.py \
+            -e PYTHON=${{ matrix.PYTHON }} \
```

Comment:
This line runs a Docker container, which we would no longer need in this case. Instead, call the `./entrypoint.sh` file directly, along with the Python version used.

```bash
run: PYTHON=3.11 ./entrypoint.sh
```

Set PYTHON=3.10 or 3.11, depending on which version of Python `ubuntu-latest` has. You should probably set it to a specific version of Ubuntu, so we know and that it won't change in the future.

Reply:
FYI, I just had a look: **Ubuntu 22** seems to ship with **Python 3.10**.
_(→ [source](https://packages.ubuntu.com/jammy/python/python3))_